src/test/java/teammates/logic/external/package-info.java

```diff
@@ -0,0 +1,4 @@
+/**
+ * Contains test cases for {@link teammates.logic.thirdparty} package.
```

Comment:
Mismatched package name

Reply:
Thanks for the comment! I originally named it as `logic.thirdparty` and did not change accordingly. I will fix it in next commit.